src/main/java/github/PullRequestServiceEx.java

```diff
@@ -0,0 +1,77 @@
+package github;
+
+import com.google.gson.reflect.TypeToken;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.egit.github.core.IRepositoryIdProvider;
+import org.eclipse.egit.github.core.client.PagedRequest;
+import org.eclipse.egit.github.core.service.PullRequestService;
+import util.HTLog;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.eclipse.egit.github.core.client.IGitHubConstants.*;
+
+public class PullRequestServiceEx extends PullRequestService {
+    private static final Logger logger = HTLog.get(PullRequestServiceEx.class);
+
+    /**
+     * Gets a pull request's review comments
+     *
+     * @param repository
+     * @param pullRequestNumber
+     * @return list of review comments
+     * @throws IOException
+     */
+    public List<ReviewComment> getReviewComments(IRepositoryIdProvider repository,
+                                                 int pullRequestNumber) throws IOException {
+        return getReviewComments(repository, Integer.toString(pullRequestNumber));
+    }
+
+    /**
+     * Gets a pull request's review comments
+     *
+     * @param repository
+     * @param pullRequestNumber
+     * @return list of review comments
+     * @throws IOException
+     */
+    public List<ReviewComment> getReviewComments(IRepositoryIdProvider repository,
```

Comment:
I was mirroring the structure of `egit`'s `getComments` interface in their service classes. The method with `int` is the commonly used one. The one with string is a fallback in case that some time in the future issue's id is not a number anymore (which is weird but still) since GitHub never guarantees that issue's id is a number.

It's the same case with page number. Although we are using numbers for getting pages from GitHub API, in rare cases, the page's number can be a hash string.